experimental-feature-windows.json

```diff
@@ -0,0 +1,8 @@
+[
+  "PSAnsiRenderingFileInfo",
+  "PSCommandNotFoundSuggestion",
+  "PSLoadAssemblyFromNativeCode",
+  "PSNativeCommandArgumentPassing",
+  "PSNativePSPathResolution",
+  "PSSubsystemPluginModel"
+]
```

Comment:
Some exp features are missing from this list. This is the list generated locally:
```
  "ExperimentalFeatures": [
    "PSAMSIMethodInvocationLogging",
    "PSAnsiRenderingFileInfo",
    "PSCleanBlock",
    "PSCommandNotFoundSuggestion",
    "PSExec",
    "PSLoadAssemblyFromNativeCode",
    "PSNativeCommandArgumentPassing",
    "PSNativeCommandErrorActionPreference",
    "PSNativePSPathResolution",
    "PSRemotingSSHTransportErrorHandling",
    "PSStrictModeAssignment",
    "PSSubsystemPluginModel",
    "Microsoft.PowerShell.Utility.PSImportPSDataFileSkipLimitCheck",
    "Microsoft.PowerShell.Utility.PSManageBreakpointsInRunspace"
  ]
```